automation/src/test/scala/org/broadinstitute/dsde/test/api/tagannotation/AuthDomainsTest.java

```diff
@@ -0,0 +1,12 @@
+package org.broadinstitute.dsde.test.api.tagannotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.scalatest.TagAnnotation;
+
+@TagAnnotation
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.METHOD, ElementType.TYPE})
+public @interface AuthDomainsTest {}
```

Comment:
Is it possible to combine these into one interface? Might help organisation and future discovery of these annotations?

Reply:
Great idea! I've refactored the annotations and group them into a higher-level `RawlsTestGroupAnnotation`.

Reply:
Oh  I see how my comment was ambiguous. I meant combine them into one file rather than many. This is a nice improvement though!

Reply:
Some updates here.

The `@interface` implicitly means `interface extends annotation.Annotation`. We can't extend `@interface` like a regular class or treat them like a regular java type. 

We can, however, place them in a non-class all in one file. The only access modifier allowed is `default` so the `@interface` must reside in the same package as the tests. I think this should be fine.

I have refactored the code according to my explanation above.